go/ir/irutil/util.go

```diff
@@ -75,7 +75,7 @@ func Vararg(x *ir.Slice) ([]ir.Value, bool) {
 
 func CallName(call *ir.CallCommon) string {
 	if call.IsInvoke() {
-		return ""
+		return call.Method.FullName()
```

Comment:
I don't know if I want to settle on these semantics. Specifically, note this:

> Note: an abstract method may be shared by multiple interfaces due to embedding; Value.Type() provides the specific interface used for this call. 

For 

```
type I1 interface {Foo()}
type I2 interface {I1}
func fn(i I2) {i.Foo()}
```

should CallName return I1.Foo or I2.Foo?

Reply:
You call `I2.Foo ()`, you don't need to know anything about the implementation of `I2`.